include/nbl/ui/CWindowManagerWin32.h

```diff
@@ -92,7 +114,9 @@ namespace nbl::ui
 			{
 				this->start();
 			}
-			~CThreadHandler() {};
+			~CThreadHandler() override
```

Comment:
this is extremely weird, thread handlers shouldnt anything virtual,
They use 100% static polymorphism (CRTP stuff)